src/GraphQL.SystemTextJson/InputsJsonConverter.cs

```diff
@@ -14,18 +14,18 @@ public class InputsJsonConverter : JsonConverter<Inputs>
     {
         /// <inheritdoc/>
         public override void Write(Utf8JsonWriter writer, Inputs value, JsonSerializerOptions options)
-            => JsonSerializer.Serialize<IReadOnlyDictionary<string, object>>(writer, value, options);
+            => JsonSerializer.Serialize<IReadOnlyDictionary<string, object?>>(writer, value, options);
 
         /// <inheritdoc/>
         public override Inputs Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
             => ReadDictionary(ref reader).ToInputs();
 
-        private static Dictionary<string, object> ReadDictionary(ref Utf8JsonReader reader)
+        private static Dictionary<string, object?> ReadDictionary(ref Utf8JsonReader reader)
```

Comment:
NSJ returns null:
```
  internal static Dictionary<string, object?>? ReadDictionary(JsonReader reader) { ...
            if (reader.TokenType == JsonToken.Null)
                return null;
```

Reply:
As far as I know, STJ only executes a JsonConverter when there is an actual object to be decoded, and never in the case of nulls.  Perhaps Newtonsoft is different.  If you suggest a change anyway, lmk.

Reply:
If so I'm fine to keep as is.